### PR TITLE
Buildingblock/paneldata

### PR DESCRIPTION
**Pull request by valerievossen**
Hey @lachlandeer, 

This is a draft of the first building block (introduction to panel data). 
For now, I added it at building-blocks/analyze-data/regression, but it might be an idea to create a new header with the other (upcoming) building blocks about fixed effects. Let me know what you think

Feedback is appreciated! In the meantime, I am working on the fixed effects building blocks (assumptions & models). Once drafts of those are finished, I will upload them in this branch as well.

### CODE REVIEW

**Comment by lachlandeer** [comment]
great -- in the mean time I'm gonna convert this to a draft PR, so I know not to merge it in until complete.

**Comment by lachlandeer** [comment]
Hey @alexandervossen, 

Do you wanna give this a first read and suggest any high level changes?

File: content/building-blocks/analyze-data/regressions/firstdifference.md
Comment (alexandervossen) [inline content/building-blocks/analyze-data/regressions/firstdifference.md:1]: I think it is important to make very clear in the first paragraph of each block when and why students should use the one or the other. As it is now it is a bit technical and does not focus so much on the situations where it should be used. Maybe you can elaborate a bit on the "Guide" component.

**Review by alexandervossen — CHANGES_REQUESTED** [review]
Great pitch @valerievossen and sorry for being so late. Let me know if you need some more clarification on the comments.

**Comment by valerievossen** [comment]
Alex and me had a quick meeting about the feedback:
- will give an intuitive example in when choosing Random Effects model is more appropriate than Fixed Effects (with data structure of multi-level fixed effects)
- First-difference and between estimator are more technical than practical to use. That's why I will keep the between estimator bb as short as it is now without giving any further example. 
- Emphasize that choice between fixed effects/random effects is the most important to make based on your data structure - FE and RE models are used the most.

**Comment by valerievossen** [comment]
Hey @alexandervossen,
All changes are made now, would appreciate another check to see if it covers your feedback!

**Review by alexandervossen — APPROVED** [review]

**Comment by alexandervossen** [comment]
> Hey @alexandervossen, All changes are made now, would appreciate another check to see if it covers your feedback!
> 
> One thing that still needs to be changed are the titles, but I will wait to see where on the website they will be ("Causal inference" content?) and then change the titles accordingly.

Great Valerie! I am not sure where to put it but I am sure @lachlandeer will know :)

**Comment by valerievossen** [comment]
Good idea! Created a new section and moved everything. I put the fixest package bb in this section as well: right after the FE model bb seemed the best place to me.

Format-wise should be ready to merge then!
@lachlandeer let me know if you want any other changes.

**Comment by shrabasteebanerjee** [comment]
just adding a note here that perhaps @srosh2000 's DiD stuff can go in this folder too?

**Comment by valerievossen** [comment]
https://tilburgsciencehub.com/building-blocks/analyze-data/regressions/impact-evaluation/ 
is that this building block you are referring to @shrabasteebanerjee? And, do you think the synthetic control bb suits under the section of panel data as well?

**Comment by hannesdatta** [comment]
> https://tilburgsciencehub.com/building-blocks/analyze-data/regressions/impact-evaluation/ is that this building block you are referring to @shrabasteebanerjee? And, do you think the synthetic control bb suits under the section of panel data as well?

added it here for now!

**Comment by hannesdatta** [comment]
This is great work & I'm merging it - it definitely 'populates' our analysis section well. 

In the future, we will have to maybe move the intro to panel data to the tutorials section - or find a better way to tie multiple building blocks together. Thanks everyone for the hard work!

**Pull request merged**